src/intent.html

```diff
@@ -347,39 +288,70 @@ <h3 id="mixing_intent_properties">Intent Properties</h3>
+     The concept name should be normalized
+     (<q>`_`</q> (U+00F5) and <q>`.`</q>  (U+002E) to <q>`-`</q> (U+002D)),
+     and compared using <a data-cite="INFRA#ascii-case-insensitive">ASCII case-insensitive</a>
+     match.  If arguments were given explicitly in the <code class="attribute">intent</code>
+     then their number gives the arity, and the fixity is determined from an explicit property
+     or may default from the concept dictionary. Otherwise, arity is assumed to be 0.</p>
+   <p>An concept is considered a <dfn id="intent_known_concept">known concept</dfn> (to the AT)
+     when the normalized name, the fixity property, and the arity
+     all match an entry in the AT's concept dictionary.
+     The speech hint in the matching entry
+     can be used be used as a guide for the generation of
+     specific audio, replacement text or braille renderings, as appropriate.
+     However, because common notations have many specialized ways of being spoken, the AT
+     is NOT constrained to use the hint as given.  For example, AT may
+     vocalize a fraction marked up with <code>&lt;mfrac></code>
+     as <q>three quarters</q> or <q>three over x</q>
+     or may vocalize an inline fraction marked up as <code>&lt;mo>/&lt;/mo></code>
+     as <q>three divided by x</q>.
+     The choice may depend on the contents
+     and carrier element associated with an
+     <code>intent="divide($num,$denom)"</code>.
+     Note that properties other than those specifying fixity
+     may also indicate different rendering choices.</p>
+   <p>Otherwise, if the concept name, fixity and arity do not match it is considered to be an
+     <dfn id="intent_unknown_concept">unknown concept</dfn> (to the AT)
+     and will be treated the same as a [=literal=];
+     that is, the name is spoken as-is after normalizing each of `-`, `_` and `.` to an inter-word space.
+     Even for an unknown concept, if a fixity property and arguments were given,
+     the speech for the arguments should be composed
+     in a manner consistent with the given fixity property, if possible.</p>
+   <p>Note that future updates of the AT and [=Intent Concept Dictionary=] may
+     include additional concepts, at which time those concepts may also receive special treatment.</p>
+
+   <p>In cases where the intent contains neither an explicit nor inferrable concept
+     the AT should generally read out the MathML in a literal or structural fashion.
+     However, any given [=properties=] should be respected if possible,
+     and may be useful to indicate the kind of mathematical object,
+     rather than giving an explicit [=concept=] name to be spoken.
+     This can be a useful technique, especially for large constructs such as tables as
+     it allows the children to be inferred without needing to be
+     explicitly referenced in the `intent` as would be the case with an <i>applicaton</i>.
+     For example, `&lt;mtable intent=":array">...` might read the table as
```

Comment:
The examples here should probably align with the core properties (even if not explicitly referencing that list) which (today at least) would be `matrix` and `system-of-equations` rather than `array` and  `aligned-equations`

Reply:
done; thanks!